lhotse/kaldi.py

```diff
@@ -7,6 +7,19 @@
 from lhotse.audio import AudioSource, Recording, RecordingSet
 from lhotse.supervision import SupervisionSegment, SupervisionSet
 from lhotse.utils import Pathlike, is_module_available
+from lhotse.audio import _audioread_info 
```

Comment:
Shall we rename `_audioread_info` to `audioread_info`
as it is used outside `audio.py`? @pzelasko

Reply:
I agree with @csukuangfj 's suggestion.